packages/story-editor/src/app/rightClickMenu/provider.js

```diff
@@ -599,20 +628,24 @@ function RightClickMenuProvider({ children }) {
 
           trackEvent('context_menu_action', {
             name: 'undo_clear_styles',
-            element: selectedElementType,
-            isBackground: selectedElement?.isBackground,
+            elements: [selectedElements.map((element) => element.type)],
```

Comment:
should this be a nested array or nah?
```suggestion
            elements: selectedElements.map((element) => element.type),
```

Reply:
good catch - this should not be a nested array